src/shared/consts.tsx

```diff
@@ -437,4 +437,6 @@ export const observerStyles = {
 
 export const enum IntegrationTypes {
   Loris = 'loris',
+  LorisUpperCase = 'LORIS', // In the API the integration type is in uppercase in some endpoints and lowercase in others
+  SampleIntegration = 'sample',
```

Comment:
What is `SampleIntegration`? If it is meant as an example please add a comment explaining usage.

Reply:
yep

Reply:
The section I mentioned to include as example is for the `configuration` param (currently declared as `SaveIntegrationParams`)